packages/aws-lambda-otel-extension/test/unit/external/helper.test.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const { expect } = require('chai');
+const { cleanResponseData } = require('../../../opt/otel-extension/external/helper');
+
+describe('external helper', () => {
+  describe('cleanResponseData', () => {
+    it('should redact entire body if it is not json', () => {
+      const data = {
+        responseData: 'This is just a standard text string',
+      };
+      const out = cleanResponseData(data);
+
+      expect(out.excludedBody).to.equal(true);
+      expect(out.responseData).to.equal('');
```

Comment:
Wouldn't it be cleaner to simply remove `responseData` property in such case? And in such case we should also not need `exludedBody` (I think this applies only where we exclude part of the response)